# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/slotmap/.cargo-checksum.json
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/slotmap/.cargo-checksum.json b/third_party/rust/slotmap/.cargo-checksum.json
new file mode 100644
--- /dev/null
+++ b/third_party/rust/slotmap/.cargo-checksum.json
@@ -0,0 +1,1 @@
+{"files":{"Cargo.toml":"93343b5fc19e795ff673b3ec726122c53df11be6a8e03f3ca91368ecee139291","src/lib.rs":"f9c4156407ca4921edfae54b1bbdca5ca004595d729d0cebd9f563edd8ca38a6"},"package":null}
\ No newline at end of file